### PR TITLE
Webdriverio Test: Run webdriverio tests suite on github actions

### PR DESCRIPTION
**Pull request by hariclerry**

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -15,6 +15,12 @@ jobs:
             - name: Create .env file
               run: "${GITHUB_WORKSPACE}/.github/scripts/environment.sh"
               shell: bash
+            - uses: actions/checkout@master
+            - uses: nanasess/setup-chromedriver@master
+            - run: |
+                  export DISPLAY=:99
```
Comment (zeljkofilipin) [inline .github/workflows/node.js.yml]: Remove this line.

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -15,6 +15,12 @@ jobs:
...
+                  chromedriver --url-base=wd/hub --port=4444 &
+                  sudo Xvfb -ac :99 -screen 0 1280x1024x24 > /dev/null 2>&1 & # optional
```
Comment (zeljkofilipin) [inline .github/workflows/node.js.yml]: Remove this line.

File: tests/selenium/wdio.conf.js [hunk 3]
```diff
@@ -13,7 +13,10 @@ function fileName( title ) {
 
 // build file path
 function filePath( test, screenshotPath, extension ) {
-	return path.join( screenshotPath, `${fileName( test.parent )}-${fileName( test.title )}.${extension}` );
+	return path.join(
+		screenshotPath,
```
Comment (sovoid) [inline tests/selenium/wdio.conf.js]: Why is there a change here?

File: tests/selenium/wdio.conf.js [hunk 4]
```diff
@@ -60,19 +63,21 @@ exports.config = {
...
-			// If DISPLAY is set, assume developer asked non-headless or CI with Xvfb.
-			// Otherwise, use --headless.
-			args: [
-				...( process.env.DISPLAY ? [] : [ '--headless' ] ),
-				// Chrome sandbox does not work in Docker
-				...( fs.existsSync( '/.dockerenv' ) ? [ '--no-sandbox' ] : [] )
-			]
+	capabilities: [
+		{
+			// For Chrome/Chromium https://sites.google.com/a/chromium.org/chromedriver/capabilities
```
Comment (sovoid) [inline tests/selenium/wdio.conf.js]: Same there should be no changes here too...is this the work of an autoformatter?

Comment (hariclerry) [reply tests/selenium/wdio.conf.js]: Yes @und3fined-v01d, I had made some changes to try to run webdriver test, however, after reverting the changes the autoformater formatted the file.

**Comment by zeljkofilipin** [comment]
Looks good in general. Two things to do before it can be merged.

- save files from tests/selenium/log
- make sure wdio.conf.js file is not chaged

**Pull request merged**